target/linux/armsr/armv8/config-6.6

```diff
@@ -113,6 +113,7 @@ CONFIG_ARM64_WORKAROUND_TSB_FLUSH_FAILURE=y
 CONFIG_ARMADA_37XX_WATCHDOG=y
 CONFIG_ARMADA_THERMAL=y
 CONFIG_ARM_ARCH_TIMER_OOL_WORKAROUND=y
+# CONFIG_ARM_CORESIGHT_PMU_ARCH_SYSTEM_PMU is not set
```

Comment:
This is already set in the generic config, see e18dda015963be53c8f4a130f20f0a7b2c2ec65e.